corehq/pillows/mappings/xform_mapping.py

```diff
@@ -43,6 +43,10 @@
             "type": "date",
             "format": DATE_FORMATS_STRING
         },
+        "server_modified_on": {
```

Comment:
yes, that's the idea. We don't have a way to reindex only one domain's data though.